test/base/TextTest.ooc

```diff
@@ -226,6 +226,17 @@ TextTest: class extends Fixture {
 			text = t"%s %s %i %i" format("test", "number", 0, 1)
 			expect(text == t"test number 0 1")
 		})
+		this add("implicit toText", func {
+			one := t"123" + 456 + 7.89f
```

Comment:
Yes but this is a standard in (almost) any programming language to be able to add floats to integers and `value = 2.5 + 6` won't surprise anyone. Certainly it is not a standard to add string objects and numbers with `operator +` like `value = "8" + 7`. What should be the result ? string `"87"` or  a number `15` ? There is no standard behaviour and this is a possible source of confusion for newcommers, becase I know people who would prefer the second option (and I've seen people implementing and using it, despite the ambiguity).
Of course I'm not asking you to skip this PR.